plugins/repository-s3/build.gradle

```diff
@@ -49,15 +49,13 @@ test {
 }
 
 thirdPartyAudit.excludes = [
-  // uses internal java api: com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderFactoryImpl
-  // uses internal java api: com.sun.org.apache.xml.internal.dtm.ref.DTMManagerDefault
-  // uses internal java api: com.sun.org.apache.xpath.internal.XPathContext
-  'com.amazonaws.util.XpathUtils',
-
   // classes are missing
   'javax.servlet.ServletContextEvent', 
   'javax.servlet.ServletContextListener', 
   'org.apache.avalon.framework.logger.Logger', 
   'org.apache.log.Hierarchy', 
   'org.apache.log.Logger',
 ]
+
+// AWS SDK is exposing some deprecated methods which we call using a delegate
```

Comment:
Methods are:

* setObjectRedirectLocation
* changeObjectStorageClass

As it's a delegate I have to implement those deprecated methods. Implementing a deprecated method will raise the exception when running gradle check tasks.
I will simply here add in the comment the concerned methods.